tests/e2e/blocks/related-articles.spec.js

```diff
@@ -1,45 +1,36 @@
 import {test, expect} from '../tools/lib/test-utils.js';
 import {publishPost, updatePost, createPostWithFeaturedImage} from '../tools/lib/post.js';
-import {
-  addCategory, addTag, addPostType,
-  removeAllPostTypes,
-} from '../tools/lib/editor.js';
+import {addCategory, addTag, addPostType, removeAllPostTypes, searchAndInsertBlock} from '../tools/lib/editor.js';
 
 test.useAdminLoggedIn();
 
-test('Test Related Posts block', async ({page, admin, editor}) => {
-  await createPostWithFeaturedImage({admin, editor}, {title: 'Test post for Related posts'});
+test('Test Related Articles block', async ({page, admin, editor}) => {
```

Comment:
It is now `Related Posts` actually! See [here](https://github.com/greenpeace/planet4-master-theme/blob/0c9ed09562f4871a850b26e8a37df78bc347949f/functions.php#L324)
We could rename the file